lnpbp-0001.md

```diff
@@ -222,68 +222,68 @@ key tweaking schemes are not usable within LN structure.
 
 Reasons: 
 - prevention of length extension attacks;
-- makes impossible to forge a message and a public key equivalent to negation of
+- impossibility to forge a message and a public key equivalent by negation of
```

Comment:
This sentence is really hard to understand. Maybe rephrase it to make it clearer? I tried my best, but not sure if it's correct now.

Reply:
I think this point can be removed altogether since it is not valid (and a leftover from earlier draft)